### PR TITLE
Separating monitor package

### PR DESCRIPTION
**Pull request by Aditevil**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Comment by Aditvil-Dev** [comment]
https://github.com/Aditvil-Dev/advisories/blob/main/neuvector-scanner.advisories.yaml

File: neuvector-monitor-5.3.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/neuvector/neuvector
+      tag: v${{package.version}}
+      expected-commit: edbdcba632835d56dcc92ba86323c8a196471289
+
+  - runs: |
+      make -C monitor
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
```
Comment (jamonation) [inline neuvector-monitor-5.3.yaml:30]: I think the binary can be installed as `/usr/bin/neuvector-monitor` - if something expects it to be `/usr/local/bin/monitor` then we can make a subpackage that creates it as a symlink.

Comment (Aditvil-Dev) [reply neuvector-monitor-5.3.yaml:30]: scanner requires it to be /usr/local/bin/monitor

File: neuvector-monitor-5.3.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      install -Dm755 monitor/monitor ${{targets.contextdir}}/usr/bin/neuvector-monitor
+
+subpackages:
+  - name: ${{package.name}}-symlink
```
Comment (jamonation) [inline neuvector-monitor-5.3.yaml:34]:
```suggestion
  - name: ${{package.name}}-compat
```

**Review by jamonation — APPROVED** [review]
CVEs have advisories so lgtm

**Pull request merged**